yellowbrick/features/rankd.py

```diff
@@ -205,12 +240,12 @@ def transform(self, X, **kwargs):
 
         Returns
         -------
-        Xp : ndarray
+        X' : ndarray
             The transformed matrix, X'
         """
         # Rank and draw the input matrix
-        Xp = self.rank(X)
-        self.draw(Xp, **kwargs)
+        R = self.rank(X)
+        self.draw(R, **kwargs)
```

Comment:
Yeh, that totally makes sense - my initial reaction was wrong and I'm glad I thought it through before responding! Thanks for updating the docstring and comments.